pallets/pallet-did-lookup/src/lib.rs

```diff
@@ -110,6 +111,12 @@ pub mod pallet {
 	#[pallet::getter(fn connected_dids)]
 	pub type ConnectedDids<T> = StorageMap<_, Blake2_128Concat, AccountIdOf<T>, ConnectionRecordOf<T>>;
 
+	/// Mapping from account identifiers to DIDs.
+	#[pallet::storage]
+	#[pallet::getter(fn connected_accounts)]
+	pub type ConnectedAccounts<T> =
+		StorageDoubleMap<_, Blake2_128Concat, DidIdentifierOf<T>, Blake2_128Concat, AccountIdOf<T>, ()>;
```

Comment:
A simple `StorageMap` is not enough, since a DID can have multiple accounts linked. So, yes, we use an empty tuple to simply mark the presence of a `(DID, Account)` tuple. Same as we did for service endpoints, where the tuple is `(DID identifier, service identifier)`.

Reply:
Could you add your explanation to the comment in the code?

Reply:
Thanks!